src/main/java/seedu/address/model/person/RsvpStatus.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Guest's RsvpStatus status in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidRsvpStatus(String)}
+ */
+public class RsvpStatus {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "RsvpStatus Status should only have one of two values. true or false.";
+
+    /*
+     * RSVP status should only consist of true or false values. Not case-sensitive.
+     */
+    public static final String VALIDATION_REGEX = "^(?i)(true|false)$";
+
+    public static final String TRUE_REGEX = "^(?i)(true)$";
+
+    public final boolean rsvpStatus;
```

Comment:
if I'm not wrong, `RsvpStatus` needs to keep track of if the guest has indicated attendance, indicated that they will not attend, or if they have not responded to the RSVP yet?